src/main/java/frc/robot/BotSubsystems.java

```diff
@@ -60,6 +62,10 @@ public static void setTeleopDefaultCommands() {
         Logger.setup("DiffDriver Teleop Default Command -> DriveDiffTank...");
         diffDriver.setDefaultCommand(BotCommands.driveDiffTank);
 
+        // DiffDriverSpark
+        Logger.setup("DiffDriverSPark Teleop Default Command -> DriveDiffTank...");
+        diffDriverSpark.setDefaultCommand(BotCommands.driveDiffTank);
```

Comment:
You need to create a new command that requires the DiffDriverSpark subsystem, and set the default command to that.